aztec-nargo/compile_then_postprocess.sh

```diff
@@ -38,17 +45,26 @@ for artifact in $artifacts_to_process; do
   job_commands=()
   for fn_index in $private_fn_indices; do
     fn_name=$(jq -r ".functions[$fn_index].name" "$artifact")
-    fn_artifact=$(jq -r ".functions[$fn_index]" "$artifact")
+    # Remove debug symbols since they don't affect vk computation, but can cause cache misses
+    fn_artifact=$(jq -r ".functions[$fn_index] | del(.debug_symbols)" "$artifact")
+    fn_artifact_hash=$(echo "$fn_artifact-$bb_hash" | sha256sum | cut -d' ' -f1)
+
+    # File to capture the base64 encoded verification key.
+    vk_cache="$CACHE_DIR/$artifact_name.verification_key_$fn_artifact_hash.vk"
+
+    # Don't regenerate if vk_cache exists
+    if [ -f "$vk_cache" ]; then
+      echo "Verification key for function $fn_name already exists"
```

Comment:
I guess users should be aware of how to reset the cache.
```suggestion
      echo "Using cached verification key for function \"$fn_name\". Delete \"$CACHE_DIR\" folder to reset the cache"
```

Reply:
Put an extra trace with a single line per file